src/utils/AppDevUtils.ts

```diff
@@ -37,8 +37,26 @@ const randomCode = (length: number): string =>
     .slice(1)
     .toUpperCase();
 
+/**
+ * Validates a possible url
```

Comment:
a possible social media handle url? maybe an example would be nice

Reply:
this util function should validate any url, so I'm gonna stray away from adding 'social media handle' for now - but word, examples are always good.